python/lsst/obs/auxTel/ingest.py

```diff
@@ -0,0 +1,129 @@
+        # jobID is the test number, and corresponds to database entries in the eTraveller/cameraTestDB
+        pathComponents = pathname.split("/")
+        if len(pathComponents) < 0:
+            raise RuntimeError("Path %s is too short to deduce raftID" % pathname)
+        sensorId, acquisitionType, testVersion, runId = pathComponents[-4:]
+
+        if phuInfo['lsstSerial'] != sensorId:
+            raise RuntimeError("Serial mismatch: the header says %s but found %s from"
+                               "path %s"%(phuInfo['lsstSerial'], sensorId, pathname))
+        phuInfo['run'] = runId  # NOT in the header
+        phuInfo['field'] = acquisitionType # NOT in the header
+        phuInfo['ccd'] = 0  # There is only one ccd in this camera
+
+        return phuInfo, infoList
+
+    def translate_wavelength(self, md):
+        """Translate wavelength provided by teststand readout.
+
+        The teststand driving script asks for a wavelength, and then reads the value back to ensure that
+        the correct position was moved to. This number is therefore read back with sub-nm precision.
+        Typically the position is within 0.005nm of the desired position, so we warn if it's not very
+        close to an integer value.
+
+        Future users should be aware that the HIERARCH MONOCH-WAVELENG key is NOT the requested value, and
+        therefore cannot be used as a cross-check that the wavelength was close to the one requested.
+        The only record of the wavelength that was set is in the original filename.
+
+        Parameters
+        ----------
+        md : `lsst.daf.base.PropertyList or PropertySet`
+            image metadata
+
+        Returns
+        -------
+        wavelength : `int`
+            The recorded wavelength as an int
+        """
+        raw_wl = md.get("MONOWL")
+        wl = int(round(raw_wl))
+        if abs(raw_wl-wl) >= 0.1:
+            logger = lsstLog.Log.getLogger('obs.auxTel.ingest')
+            logger.warn(
+                'Translated significantly non-integer wavelength; %s is more than 0.1nm from an integer value', raw_wl)
+        return wl
+
+    def translate_visit(self, md):
+        """Generate a unique visit from the timestamp
+
+        It might be better to use the 1000*runNo + seqNo, but the latter isn't currently set
+
+        Parameters
+        ----------
+        md : `lsst.daf.base.PropertyList or PropertySet`
+            image metadata
+
+        Returns
+        -------
+        visit_num : `int`
+            Visit number, as translated
+        """
+        mjd = md.get("MJD-OBS")
+        mmjd = mjd - 55197              # relative to 2010-01-01, just to make the visits a tiny bit smaller
+        return int(1e5*mmjd)            # 86400s per day, so we need this resolution
+
+##############################################################################################################
+
+
+class AuxTelCalibsParseTask(CalibsParseTask):
```

Comment:
It would be great to have sphinx docstrings in this class too.  I understand that they are `_` methods, but we do that with `_` methods elsewhere.